bigquery_etl/format_sql/format.py

```diff
@@ -43,18 +54,22 @@ def _format_path(check, path):
             fully_referenced_query = query
     except NotImplementedError:
         fully_referenced_query = query  # not implemented for scripts
+    except ParseError:
+        print(f"Invalid syntax found for: {path}")
```

Comment:
If the parse error indicates what the actual syntax error is then it'd be helpful to surface that, like maybe
```suggestion
    except ParseError as e:
        print(f"Invalid syntax found for {path}: {e}")
```